### ISSUE TITLE
Load game

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/djog_unos_2018#97]
Depends on:

 * [x] #583 

**Is your feature request related to a problem? Please describe.**

After #95 ('A game can be saved') it follows that a game can also be loaded.

**Describe the solution you'd like**

This test should pass:

```c++
  #define FIX_ISSUE_97
  #ifdef FIX_ISSUE_97
  // A game can be loaded
  {
    const game g(create_test_default_tiles(),
                 std::vector<agent>{agent(agent_type::spider, 0, 0, 100)}
                );
    const std::string filename{"tmp"};
    const QString actual_path = QString::fromStdString(SAVE_DIR) + filename.c_str() + ".sav";
    if (QFile::exists(filename.c_str()))
    {
      std::remove(filename.c_str());
    }
    assert(!QFile::exists(filename.c_str()));
    save(g, filename);
    assert(QFile::exists(actual_path));
    const game h = load(filename);
    assert(g == h);
  }
  #endif // FIX_ISSUE_97
```

**Describe alternatives you've considered**

None.

**Additional context**

Will do so after #95 is done.

**Comment by richelbilderbeek** [comment]
Re-opened this Issue

**Comment by Joshua260403** [comment]
Oi mate wtf are you doing? I was doing the liad screen and this very much depends on that...

**Comment by Joshua260403** [comment]
What exactly did you do @Rijk-van-Putten

**Comment by Joshua260403** [comment]
I see he did nothing, so I'm assuming this was closed because it is outdated and duplicate of #461
